docs/data-management/metrics/simple-metric.md

```diff
@@ -148,6 +148,28 @@ For example, consider a podcast app with a fact that logs **podcast listen event
 * _Episodes listened per user_, to measure overall content consumption. This can be captured correctly with a `Count` metric. Since repeated listens to the same episode are rare, using `Count Distinct` on episode id would give similar results but incur higher warehouse costs.
 :::
 
+#### Last Value
+
+Last Value computes the most recent value of a fact. It can be interpreted as averages across entities:
+
+$\frac{\text{SUM of most recent fact value}}{\text{Number of unique entities assigned}}$
+
+In SQL:
+
+```
+select 
+  <entity_id>, 
+  last_value(<fact_col>) over (partition by <entity_id> order by <fact_timestamp>)
+from ... 
+```
+
+Examples:
```

Comment:
Here and below is essentially plagiarlzed from https://updates.eppo.cloud/en

Reply:
Not sure if both of these places should share copy / language style -- open to thoughts here!